script-test/playbackstrategies/msestrategytest.js

```diff
@@ -106,66 +111,77 @@ require(
 
           expect(mseStrategy.transitions.canBePaused()).toBe(true);
         });
+      });
 
-        it('canBeginSeek() Transition is true', function () {
+      describe('Load', function () {
+        it('should create a video element and add it to the media element', function () {
           setUpMSE();
 
-          expect(mseStrategy.transitions.canBeginSeek()).toBe(true);
+          expect(playbackElement.childElementCount).toBe(0);
+
+          mseStrategy.load('src', null, 0);
+
+          expect(playbackElement.firstChild).toBe(mockMediaElement);
+          expect(playbackElement.childElementCount).toBe(1);
         });
-      });
 
-      describe('Load', function () {
-        it('should create a video element and add it to the video container', function () {
-          setUpMSE();
+        it('should create an audio element and add it to the media element', function () {
+          setUpMSE(null, null, MediaKinds.AUDIO);
 
-          expect(videoContainer.childElementCount).toBe(0);
+          expect(playbackElement.childElementCount).toBe(0);
 
           mseStrategy.load('src', null, 0);
 
-          expect(videoContainer.firstChild).toBe(mockVideoElement);
-          expect(videoContainer.childElementCount).toBe(1);
+          expect(playbackElement.firstChild).toBe(mockMediaElement);
+          expect(playbackElement.childElementCount).toBe(1);
+        });
+
+        it('canBeginSeek() Transition is true', function () {
```

Comment:
I think you've accidentally moved this test out of the `'transitions'` describe block.